proto/profile.proto

```diff
@@ -1,3 +1,17 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
What was the year in the previous version of the license header? Use that?

Reply:
Other than the accidentally added "// Copyright 2016 Google Inc.", we've never had any year here (I looked through past commits, and didn't see any apache header files)

I'm going to use 2016, I guess.